csharp/sources/Valkey.Glide/Internals/Request.SortedSetCommands.cs

```diff
@@ -64,4 +64,31 @@ public static Cmd<long, long> SortedSetAddAsync(ValkeyKey key, SortedSetEntry[]
 
         return Simple<long>(RequestType.ZAdd, [.. args]);
     }
+
+    public static Cmd<long, bool> SortedSetRemoveAsync(ValkeyKey key, ValkeyValue member, CommandFlags flags = CommandFlags.None)
+    {
+        if (flags != CommandFlags.None)
+        {
+            throw new NotImplementedException("Command flags are not supported by GLIDE");
+        }
+
+        GlideString[] args = [key.ToGlideString(), member.ToGlideString()];
+        return new(RequestType.ZRem, args, false, response => response == 1);
```

Comment:
Pull in the new changes from main if needed. There is a new method that allows you to do 
```suggestion
        return Boolean<long>(RequestType.ZRem, args);
```

Reply:
Updated.

Reply:
Do you miss an commit?

Reply:
It got removed during a rebase. Updating.